mautic/mautic
# [UI] Interactive filters for tables

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

Visual UI for quick filters.

![image](https://github.com/user-attachments/assets/b2a3864d-a6e4-475c-b319-1dc34418aaf6)
![image](https://github.com/user-attachments/assets/00e3e538-fa0f-4707-ba94-bce6c78d1bfa)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > Create some > Open the filters and play with each
3. Repeat for other resources if you want
